.teamcity/src/subprojects/release/publishing/PublishMavenBuild.kt

```diff
@@ -0,0 +1,29 @@
+package subprojects.release.publishing
+
+import jetbrains.buildServer.configs.kotlin.v10.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import subprojects.build.core.*
+import java.lang.RuntimeException
+
+class PublishMavenBuild(private val build: Build) : BuildType({
+    id("KtorPublishMavenBuild_${build.name}".toExtId())
+    name = "Publish ${build.name} to Maven"
+
+    dependencies {
+        val buildId = build.build?.id
+        if (buildId != null) {
+            artifacts(buildId) {
+                buildRule = lastSuccessful()
+                artifactRules = stripReportArtifacts(artifactRules)
+            }
+        } else {
+            throw RuntimeException("Build ID not found for entry ${build.name}")
+        }
+
+    }
+})
+
+fun stripReportArtifacts(artifacts: String): String {
```

Comment:
`stripReportArtifacts` name is too abstract and doesn't covey what it does.

Reply:
In what sense? It's dropping the reporting artifacts from the artifact list.

Reply:
Then `remove` is the better name. The problem here is again that you use terms for strings.

Reply:
What is a term for string? Strip? It has nothing to do with strings. Strip can be applied to anything.